controls/srg_ctr/SRG-APP-000400-CTR-000960.yml

```diff
@@ -4,7 +4,7 @@ controls:
   - medium
   title: {{{ full_name }}} must prohibit the use of cached authenticators after
     an organization-defined time period.
```

Comment:
Do we need a variable so that people can tweak this to align with the STIG guidance?

Reply:
Nice catch.
I thought rule `oauth_or_oauthclient_token_maxage` was checking for value `28800` as the rule description states. But, checking again, I see that the rule's check rely on two checks from other rules (`oauth_token_maxage` and `oauthclient_token_maxage`) and they don't check for a specific value.

Reply:
I have parametrized the relevant rules and customized the variable in the OCP4 STIG profile.